gudpy/gui/widgets/core/main_window.py

```diff
@@ -600,26 +605,72 @@ def loadInputFile_(self):
                                      "GudPy Error",
                                      "Could not open file")
 
+    def loadProject_(self):
+        """Load from previous GudPy project
+        """
+        projectDir = QFileDialog.getExistingDirectory(self, 'Select Project')
+
+        try:
+            self.gudrunFile = GudrunFile(
+                projectDir=projectDir, format=Format.YAML)
+            autosave = self.tryLoadAutosaved(self.gudrunFile.path)
```

Comment:
Perhaps the variable name could be a little more descriptive? Unless I'm misunderstanding the role here!
```suggestion
            autosavedFile = self.tryLoadAutosaved(self.gudrunFile.path)
```